core/app/help.go

```diff
@@ -23,6 +23,45 @@ import (
 	"github.com/google/gapid/core/app/flags"
 )
 
+// MakeDocs
+func MakeDoc(ctx context.Context, name string) {
+	filename := name + ".md"
+	w, _ := os.Create(filename)
+	globalVerbs.selected = nil
+	fmt.Fprintln(w, "**Note**: Autogenerated documentation, do not modify!")
+	fmt.Fprintln(w, "# GAPIT Help")
+
+	// Print list of verbs
+	fmt.Fprintln(w, "| Command | Short help |")
+	fmt.Fprintln(w, "| -- | -- |")
+	format := fmt.Sprintf("|[%%s](#%%s)|%%s|")
```

Comment:
this is just a constant string, no need to send it through sprintf.

Reply:
Fixed.